pkg/cmd/openshift-apiserver/openshiftapiserver/config.go

```diff
@@ -125,6 +125,15 @@ func NewOpenshiftAPIConfig(config *openshiftcontrolplanev1.OpenShiftAPIServerCon
 		return nil, err
 	}
 	authenticationOptions := genericapiserveroptions.NewDelegatingAuthenticationOptions()
+	authenticationOptions.ClientCert.ClientCA = config.ServingInfo.ClientCA
```

Comment:
was this not wired?!

Reply:
> was this not wired?!

nope.  The server was configured with one and a different one was sent for authentication.  I think we made them both agree eventually.